packages/reassure-cli/src/utils/git.ts

```diff
@@ -1,4 +1,5 @@
 import simpleGit from 'simple-git';
+import * as Logger from './logger';
```

Comment:
I am confused here, why are we using `* as Logger` if the util exports `{ logger }` as a named export?

Reply:
Especially since in the very same package we're doing a named import instead already.

Reply:
Local `logger` is `reassure-logger` but bound to package prefix like `reassure-cli`.

Reply:
good catch!

Reply:
fixed